imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import unicode_literals
+from .models import Photo, Album
+from django.contrib.auth.models import User
+from django.test import TestCase
+import factory
+from faker import Faker
+
+fake = Faker()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = factory.LazyAttribute(lambda a: '{}.{}@example.com'.format(
+                                  a.first_name, a.last_name).lower())
+    username = factory.LazyAttribute(lambda a: '{}{}{}{}'.format(
+                                     a.last_name[:1], a.first_name[1:],
+                                     a.first_name[:1], a.last_name[1:]).lower())
```

Comment:
The style of indentation here is bothering me a bit.  It's awkward.  Try something like this instead:

```python
    email = factory.LazyAttribute(
        lambda a: '{}.{}@example.com'.format(a.first_name, a.last_name).lower()
    )
    username = factory.LazyAttribute(
        lambda a: '{}{}{}{}'.format(
            a.last_name[:1], a.first_name[1:], a.first_name[:1], a.last_name[1:]
        ).lower()
    )
```